src/main/java/net/minecraft/block/HoneyBlock.java

```diff
@@ -99,6 +103,11 @@ private void triggerSlideDownBlock(Entity entity, BlockPos pos)
 
     private void setSlideVelocity(Entity entity)
     {
+        Module noSlow = Client.getInstance().moduleManager.getModuleByClass(NoSlow.class);
+        if (entity instanceof ClientPlayerEntity && noSlow.isEnabled2() && noSlow.getBooleanValueFromSettingName("Blocks")) {
+            entity.fallDistance = 0;
+            return;
+        }
```

Comment:
```suggestion
        // MODIFICATION BEGIN: cancel slide velocity if NoSlow is enabled, this is the local player, and Blocks is enabled in NoSlow.
        Module noSlow = Client.getInstance().moduleManager.getModuleByClass(NoSlow.class);
        if (entity instanceof ClientPlayerEntity && noSlow.isEnabled2() && noSlow.getBooleanValueFromSettingName("Blocks")) {
            entity.fallDistance = 0;
            return;
        }
        // MODIFICATION END
```
it's kinda like what paper or whatever other Minecraft server forks do, but it's pretty useless unless you're going to rewrite it to use Mixins (yapping begin: will probably try to do that once I get Fabric loom working with Searge mappings, because I don't feel like find & replace is the right solution, I'll probably get it on Yarn mappings once we finally migrate over to Fabric, not sure if some other people won't like it, but I think it has way more upsides than downsides, now yapping ends).